Contribute/Pull-Requests/index.md

```diff
@@ -58,4 +58,4 @@ And that is all you need to do to create a fork, sync it and make a pull request
 ### Step 4: Wait for an action
 
 Hopefully your work will be merged immediately.  It might happen that your pull request receives a comment and a *request for changes*. We hope you are able to work with us to update your PR so we can merge it in!
-Sometimes a label is added to a PR.  We have described a [list of different labels we often use](../github-issues.md).
+Sometimes a label is added to a PR.  We have described a [list of different labels we often use](https://github.com/umbraco/UmbracoDocs/labels).
```

Comment:
```suggestion
Sometimes a label is added to a PR.  We have described a [list of different labels we often use](../index.md#labels).
```

It is actually this section we wanted to refer to: https://our.umbraco.com/documentation/Contribute/#labels 

Used to be a standalone article and then some things were merged, missed that link though.